request-manager/graph/grapher.go

```diff
@@ -149,19 +149,15 @@ func (gr *Grapher) CheckSequences() (seqGraphs map[string]*Graph, seqErrors map[
 					if len(missingSets) > 1 {
 						multiple = "s"
 					}
-					seqErrors[seqName] = fmt.Errorf("node%s did not set job args declared in 'sets': %s", multiple, strings.Join(msg, "; "))
+					seqErrors[seqName] = append(seqErrors[seqName], fmt.Errorf("node%s did not set job args declared in 'sets': %s", multiple, strings.Join(msg, "; ")))
 				}
 			}
 		}
 
 		// If we were unable to check any sequences, there must be a cyclical dependency.
 		if !newSeqChecked {
 			for seqName, _ := range seqsToCheck {
-				// This overwrites a build error, if there was one. We want all
-				// sequences that were part of the cyclical dependency to have
-				// this error; otherwise, we imply that some sequence wasn't part
-				// of the cyclical dependency when it actually was.
-				seqErrors[seqName] = fmt.Errorf("part of cyclical dependency among sequences")
+				seqErrors[seqName] = append(seqErrors[seqName], fmt.Errorf("part of cyclical dependency among sequences"))
```

Comment:
Can we report which other seq form the cycle? Would that be seqName in cycle with seqsToCheck[*] - seqName ?